neqo-transport/src/cc/cubic.rs

```diff
@@ -133,23 +133,23 @@ impl Cubic {
     /// K = cubic_root((W_max - W_cubic) / C / MSS);
     ///
     /// <https://www.rfc-editor.org/rfc/rfc8312#section-4.1>
-    fn calc_k(&self, curr_cwnd: f64, max_datagram_size: usize) -> f64 {
-        ((self.w_max - curr_cwnd) / CUBIC_C / convert_to_f64(max_datagram_size)).cbrt()
+    fn calc_k(&self, curr_cwnd_f64: f64, max_datagram_size_f64: f64) -> f64 {
```

Comment:
```suggestion
    fn calc_k(&self, curr_cwnd: f64, max_datagram_size: f64) -> f64 {
```

Given that there is no non-f64 in scope here, I don't think the suffix is necessary.